app/src/pages/playground/PlaygroundOutput.tsx

```diff
@@ -119,7 +137,9 @@ export function PlaygroundOutput(props: PlaygroundOutputProps) {
       variant="compact"
       bodyStyle={{ padding: 0 }}
     >
-      <View padding="size-200">{OutputEl}</View>
+      <View padding="size-200">
+        <MarkdownDisplayProvider>{OutputEl}</MarkdownDisplayProvider>
```

Comment:
ahh good question, about if they should share, I think you might run into some weird edge cases if the all are different since the markdown display preference is persisted in local storage, see preferencesStore. fwiw on traces the state is shared for better or worse with no complaints to my knowledge

Reply:
would probably just do whats done on traces for a first pass

Reply:
Based on the implementation of MarkdownDisplayProvider, it appears that I can have many disconnected trees of MarkdownDisplayProvider and they will all share state since they pull from the same zustand store in PreferencesContext

So it they already share state, false alarm